core/variant/callable_bind.cpp

```diff
@@ -184,25 +184,25 @@ CallableCustomUnbind::~CallableCustomUnbind() {
 
 Callable callable_bind(const Callable &p_callable, const Variant &p_arg1) {
 	const Variant *args[1] = { &p_arg1 };
-	return p_callable.bind(args, 1);
+	return p_callable.bindv(args, 1);
 }
 
 Callable callable_bind(const Callable &p_callable, const Variant &p_arg1, const Variant &p_arg2) {
 	const Variant *args[2] = { &p_arg1, &p_arg2 };
-	return p_callable.bind(args, 2);
+	return p_callable.bindv(args, 2);
 }
 
 Callable callable_bind(const Callable &p_callable, const Variant &p_arg1, const Variant &p_arg2, const Variant &p_arg3) {
 	const Variant *args[3] = { &p_arg1, &p_arg2, &p_arg3 };
-	return p_callable.bind(args, 3);
+	return p_callable.bindv(args, 3);
 }
 
 Callable callable_bind(const Callable &p_callable, const Variant &p_arg1, const Variant &p_arg2, const Variant &p_arg3, const Variant &p_arg4) {
 	const Variant *args[4] = { &p_arg1, &p_arg2, &p_arg3, &p_arg4 };
-	return p_callable.bind(args, 4);
+	return p_callable.bindv(args, 4);
 }
 
 Callable callable_bind(const Callable &p_callable, const Variant &p_arg1, const Variant &p_arg2, const Variant &p_arg3, const Variant &p_arg4, const Variant &p_arg5) {
```

Comment:
These look like something that can be removed now. There is only 9 uses of this.